### ISSUE TITLE
Can not build com.gog.Base: udev compilation error

### ISSUE THREAD

**Issue by V02460** [kujeger/flatpak-gog#10]
When trying to compile `com.gog.Base` with glibc 2.29, udev fails to build and these three error messages appear all over the place:
```
   undefined reference to `major'
   undefined reference to `minor'
   undefined reference to `makedev'
```

It seems related to [this glibc issue](https://sourceware.org/bugzilla/show_bug.cgi?id=19239).

To fix, it might be enough to use a newer version of udev, where `#include <sys/sysmacros.h>` should be added properly.

**Comment by monomadic** [comment]
I have this same error... couldn't get any further.

**Comment by kujeger** [comment]
Should be fixed by #11
